netlify-plugin-cloudinary/tests/on-build.test.js

```diff
@@ -1,8 +1,20 @@
-import { vi, expect, describe, test, beforeEach,afterEach } from 'vitest';
 import { promises as fs } from 'fs';
+import path from 'node:path';
+import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest';
 import { onBuild } from '../src/';
 import { ERROR_API_CREDENTIALS_REQUIRED } from '../src/data/errors';
 
+const contexts = [
+  {
+    name: 'production',
+    url: 'https://netlify-plugin-cloudinary.netlify.app'
+  },
+  {
+    name: 'deploy-preview',
+    url: 'https://deploy-preview-1234--netlify-plugin-cloudinary.netlify.app'
+  }
+]
+
```

Comment:
For deploy-preview, I noticed you were using the DEPLOY_PRIME_URL instead. I'd be happy to readd that, just let me know~